### PR TITLE
feat: add kusama currencies

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: parachain/src/chain_spec.rs [hunk 1]
```diff
@@ -104,7 +104,7 @@ pub fn local_config(id: ParaId) -> ChainSpec {
             serde_json::from_value(json!({
                 "ss58Format": 42,
                 "tokenDecimals": [10, 8],
-                "tokenSymbol": ["DOT", "interBTC"]
+                "tokenSymbol": ["KSM", "kBTC"]
```
Comment (sander2) [inline parachain/src/chain_spec.rs:107]: I think this kBTC renaming decision hasn't been finalized yet, right?

Comment (sander2) [reply parachain/src/chain_spec.rs:107]: Is KINT not required here?

Comment (gregdhill) [reply parachain/src/chain_spec.rs:107]: I think kBTC has been finalized, it was just iBTC that is still undecided (cc: @alexeiZamyatin).

Comment (gregdhill) [reply parachain/src/chain_spec.rs:107]: Yeah we can add KINT here too.

**Review by sander2 — APPROVED** [review]

**Pull request merged**